hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -95,6 +95,43 @@
       width:   "1"
     }
   ],
+  features: [
+    { name: "AON_TIMER.WAKEUP",
+      desc: '''
+            The AON_TIMER tick period can be configured.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.BARK",
+      desc: '''
+            The AON_TIMER block has a watchdog that can bark if not petted in the programed time.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.BITE",
+      desc: '''
+            The AON_TIMER block has a watchdog that can bite causing a reset if not petted in the programed time.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.PAUSE",
+      desc: '''
+            The AON_TIMER block will pause the watchdog when the heart is in sleep mode.
+            '''
+    },
+    { name: "AON_TIMER.WATCHDOG.DISABLE_BY_LC_CTRL",
+      desc: '''
+            The AON_TIMER block will disable the watchdog when the life cycle controller escalates.
+            '''
+    },
+    { name: "AON_TIMER.INTERRUPT.WATCHDOG",
+      desc: '''
+            The AON_TIMER block can generate interruptions when the watchdog bark threashold is reached.
```

Comment:
I think the usual phrase is "generate interrupts"

Also, spelling of "threshold"

Reply:
Fixed.